test/models/orchestration/dhcp_test.rb

```diff
@@ -239,9 +239,9 @@ def teardown
     end
     h.ip = h.ip.succ
     assert h.valid?
-    # 1st is creation from factory, 2nd is triggered by h.valid?
-    assert_equal 2, h.queue.items.count {|x| x.action == [ h.primary_interface, :set_dhcp ] }
-    # and also one deletion (of original creation)
+    # TODO replace this with h.queue.find_by_action - I want these changes to be more explicit for the initial review,
```

Comment:
I guess you can now replace it :-) makes sense to me

Reply:
Sure, anything else before I rebase?

Reply:
@ares can you confirm that the changes in this test (2 -> 1) are correct? I want just double check this, I don't see any reason to have these, I think it was simply verified because of the design.

Reply:
I will implement "IDs" (see initial description) which will make this much easier. Matching via actions is not possible because NIC/Host are not kept in memory for whole time.

Reply:
I believe this is correct, I believe I bumped those numbers just because `valid?` added new item to queue